gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/connection.js

```diff
@@ -199,6 +218,22 @@ class Connection extends EventEmitter {
     });
   }
 
+   _getSessionRequest(id, op, args) {
+     if (args) {
+       args = this._adaptArgs(args, true);
+     } else {
+       args = {'gremlin': 'session.close()'};
+     }
+     args['session'] = this._sessionId;
+ 
+     return ({
+       'requestId': { '@type': 'g:UUID', '@value': id },
+       'op': op || 'eval',
+       'processor': 'session',
+       'args': args
+     });
+   }
```

Comment:
Maybe we could extract common functionality between `_getRequest()` and `_getSessionRequest()`.